### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Estaffieri**
## Description
This  PR adds testing with Jest and react testing library to our project.

## Motivation and Context
We were aiming for 25% coverage and were initially planning on working with Cypress, but the integration proved more difficult than anticipated.

We want to test to ensure that the results from our components returned as expected.

Tests are not currently all passing.

## How Has This Been Tested?
Running ```npm test``` in the terminal

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by brycemara** [comment]
Tricky testing but I'm happy we got some done.

File: src/Overview/Overview.test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import "react-native-gesture-handler";
+import "@testing-library/jest-dom/extend-expect";
+import reactTestRenderer from "react-test-renderer";
+import { render, fireEvent } from "@testing-library/react-native";
+import App from "../App/App";
+import HomePage from "../HomePage/HomePage";
+import Overview from "./Overview";
+
+describe("Overview", () => {
+  it.skip("should render correctly", () => {
```
Comment (Codo-Baggins) [inline src/Overview/Overview.test.js:11]: This test is currently being skipped!  We should unskip this before merging!

**Review by Codo-Baggins — COMMENTED** [review]
This is tough, good attempt here!

File: src/HomePage/HomePage.test.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    },
+  ],
+};
+describe("HomePage", () => {
+  it("should render correctly", () => {
+    const { getByTestId } = render(<HomePage />);
+    const homeImage = getByTestId("homeFront");
+
+    expect(homeImage.props.source).toBe(
+      require("../../assets/backdrops/homefront.png")
```
Comment (cameronRomo) [inline src/HomePage/HomePage.test.js:50]: Nice way of rendering the image.

**Pull request merged**